app/views/includes/head.njk

```diff
@@ -1 +1,12 @@
 <link href="{{ css_path }}" media="screen" rel="stylesheet" type="text/css" />
+
+<!-- Global site tag (gtag.js) - Google Analytics -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=UA-142639139-3"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
```

Comment:
I realise this is probably copy-pasted, but we should stick with our JS lint conventions, namely no semicolons and more space padding in the function declaration, which would mean:
```
window.dataLayer = window.dataLayer || []
function gtag () {
  dataLayer.push(arguments)
}
gtag('js', new Date())

```